enterprise/core-edge/src/main/java/org/neo4j/coreedge/catchup/storecopy/LocalDatabase.java

```diff
@@ -140,9 +138,24 @@ public void delete() throws IOException
 
     public boolean isEmpty() throws IOException
     {
-        // TODO: Below doesn't work for an imported store. Need to check high-ids as well.
-        ReadOnlyTransactionIdStore readOnlyTransactionIdStore = new ReadOnlyTransactionIdStore( pageCache, storeDir );
-        return readOnlyTransactionIdStore.getLastCommittedTransactionId() <= BASE_TX_ID;
+        return !hasStoreFiles();
+    }
+
+    private boolean hasStoreFiles()
+    {
+        for ( StoreType storeType : StoreType.values() )
+        {
+            StoreFile storeFile = storeType.getStoreFile();
+            if(storeFile != null)
+            {
+                boolean exists = new File( storeDir,storeFile.storeFileName() ).exists();
```

Comment:
Can you use FileSystemAbstraction.fileExists() here instead of File.exists()? Maybe you actually want to delegate to the StoreFiles type - you have one of those as a field and it already has a FileSystemAbstraction.